src/normalizer/chinese.rs

```diff
@@ -18,7 +18,7 @@ impl Normalizer for ChineseNormalizer {
         options: NormalizerOption,
     ) -> Box<dyn Iterator<Item = Token<'o>> + 'o> {
         let mut lemma = String::new();
-        let mut char_map = if options.create_char_map { Some(Vec::new()) } else { None };
+        let mut char_map = options.create_char_map.then_some(Vec::new());
```

Comment:
Sorry @crudiedo, I made a mistake 😓 `then` is better in this case because it lazily calls `Vec::new()` if the boolean is true:
```suggestion
        let mut char_map = options.create_char_map.then(Vec::new);
```

my bad! 😅

Reply:
NP! :)